src/utils/fiatToCrypto.js

```diff
@@ -23,20 +23,31 @@ import { getEnv } from 'configs/envConfig';
 
 import type { AltalixTrxParams, SendwyreRates, SendwyreTrxParams } from 'models/FiatToCryptoProviders';
 import type SDKWrapper from 'services/api';
+import { RAMP_CURRENCY_TOKENS } from '../constants/assetsConstants';
 
-export function rampWidgetUrl(address: string, email?: string, plrMode?: boolean) {
+const PILLAR = 'Pillar';
+
+export function rampWidgetUrl(
+  address: string,
+  email?: string,
+  fiatCurrency: string,
+  fiatValue: string,
+) {
+  const values = RAMP_CURRENCY_TOKENS.join(',');
   const params = {
-    swapAsset: plrMode ? 'PLR' : null, // This turns on the ability to purchase PLR
+    hostAppName: PILLAR,
+    fiatCurrency,
+    fiatValue,
     hostApiKey: getEnv().RAMPNETWORK_API_KEY,
     userAddress: address,
     ...(email ? { userEmailAddress: email } : {}),
+    swapAsset: values,
   };
 
-  return `${getEnv().RAMPNETWORK_WIDGET_URL}?${querystring.stringify(params)}`;
+  return `${getEnv().RAMPNETWORK_WIDGET_URL}?${querystring.stringify(params).replace(/%2C/g, ',')}`;
```

Comment:
It's worth putting a comment here into why you're replacing %2C with a comma...

Reply:
%2c should be space, we should probably make the replacement earlier on and just for certain field.

Reply:
No, %2C is comma. So when we join(','), array has comma. When we stringify comma is replaced with /%2C

Reply:
Either way replacing reserved character (like ',') feels hackings. According to querystring library docs, the library should be able to serialize list parameters if passed as string array, i.e. 
```
swapAsset: RAMP_CURRENCY_TOKENS,
```

That would be a preferred solution.

Reply:
Just to clear up a possible misunderstanding here - whatever solution we put into place, the end result for the swapAsset array needs to result in this:

`https://buy.ramp.network/?hostAppName=Pillar&fiatCurrency=GBP&fiatValue=100&swapAsset-ETH,DAI,MATIC,USDC,USDT`

(hostApiKey, userAddress params removed)

The Ramp service doesn't accept / understand a URL encoded comma separated values.